tutorials/hist/ReadROOTFile.C

```diff
@@ -0,0 +1,26 @@
+/// \file
+/// \ingroup tutorial_hist
+/// \notebook-js
+/// Read a histogram from a ROOT File
+///
+/// This tutrial demonstrates how a Histogram
+/// can be read from a ROOT File.
+/// See CreateROOTFile.C to see how one can be created.
+///
+/// To run this example, first run CreateROOTFile.C and then
+/// run this file.
+///
+/// \macro_image
+/// \macro_code
+///
+/// \author Advait Dhingra
+
+void ReadROOTFile()
+{
+   TFile *f = new TFile("demo.root");
```

Comment:
`"hsimple.root"` should always exist

Reply:
Please use a combination of `gROOT->GetTutorialsDir()` and `hsimple.root` - which does exist - instead of `demo.root` - which doesn't exist. You will have to use one of the histograms from `hsimple.root` then, of course.